app/models/annotation_text.rb

```diff
@@ -14,4 +14,20 @@ def escape_content
     content.gsub('\\', '\\\\\\') # Replaces '\\' with '\\\\'
            .gsub(/\r?\n/, '\\n')
   end
+
+  def get_stats
+    applications = self.annotations.joins(result: { grouping: :group })
```

Comment:
Add in a `TODO` to simplify the second `joins` after we make `creator` a non-polymorphic association.